src/date_input.jsx

```diff
@@ -129,8 +127,8 @@ var DateInput = React.createClass({
     }
 
     return <input
-        ref='input'
-        type='text'
+        ref="input"
```

Comment:
Interesting change here as the **nui-widgets** eslint config is the exact opposite -- it complains about double quotes and requires single quotes.